gensrc/thrift/FrontendService.thrift

```diff
@@ -581,9 +608,14 @@ service FrontendService {
     MasterService.TMasterResult finishTask(1:MasterService.TFinishTaskRequest request)
     MasterService.TMasterResult report(1:MasterService.TReportRequest request)
     MasterService.TFetchResourceResult fetchResource()
+    
+    // those three method are used for asynchronous mini load which will be abandoned
     TFeResult miniLoad(1:TMiniLoadRequest request)
     TFeResult updateMiniEtlTaskStatus(1:TUpdateMiniEtlTaskStatusRequest request)
     TFeResult loadCheck(1:TLoadCheckRequest request)
+    // this method is used for streaming mini load
+    TMiniLoadBeginResult miniLoadBegin(TMiniLoadBeginRequest request)
+    TFeResult isMiniLoadStreaming()
```

Comment:
I think you can make it more common function. like if we add some other functional, we can reuse this function.

So you can pass function in which is want to be checked

Reply:
OK， I will change it to another function